### ISSUE TITLE
Add support for default datacenter

### ISSUE THREAD

**Issue by miklinux** [WeAreFarmGeek/diplomat#217]
It would be really helpful if the default datacenter could be set at configuration time
```
# Initialize Consul
Diplomat.configure do |config|
   config.datacenter = 'mydc'
end
```
and all API calls would use that each time, unless overridden.

**Comment by pierresouchay** [comment]
@miklinux Yes, that's a good idea, would you like to propose a PR, I'll be glad to review it

**Comment by miklinux** [comment]
@pierresouchay Will look into it

**Comment by miklinux** [comment]
Hi @pierresouchay,
here's the PR for reviewing!

https://github.com/WeAreFarmGeek/diplomat/pull/220
